crypto/s2n_composite_cipher_aes_sha.c

```diff
@@ -20,72 +20,70 @@
 #include "crypto/s2n_cipher.h"
 #include "crypto/s2n_fips.h"
 #include "crypto/s2n_openssl.h"
-
 #include "tls/s2n_crypto.h"
-
-#include "utils/s2n_safety.h"
 #include "utils/s2n_blob.h"
+#include "utils/s2n_safety.h"
 
 /* LibreSSL and BoringSSL support the cipher, but the interface is different from Openssl's. We
  * should define a separate s2n_cipher struct for LibreSSL and BoringSSL.
  */
 #if !defined(LIBRESSL_VERSION_NUMBER) && !defined(OPENSSL_IS_BORINGSSL)
-/* Symbols for AES-SHA1-CBC composite ciphers were added in Openssl 1.0.1
+    /* Symbols for AES-SHA1-CBC composite ciphers were added in Openssl 1.0.1
  * These composite ciphers exhibit erratic behavior in LibreSSL releases.
  */
-#if S2N_OPENSSL_VERSION_AT_LEAST(1,0,1)
-#define S2N_AES_SHA1_COMPOSITE_AVAILABLE
-#endif
-#if defined(AWSLC_API_VERSION) && (AWSLC_API_VERSION <= 17)
-#undef S2N_AES_SHA1_COMPOSITE_AVAILABLE
-#endif
-/* Symbols for AES-SHA256-CBC composite ciphers were added in Openssl 1.0.2
+    #if S2N_OPENSSL_VERSION_AT_LEAST(1, 0, 1)
+        #define S2N_AES_SHA1_COMPOSITE_AVAILABLE
+    #endif
+    #if defined(AWSLC_API_VERSION) && (AWSLC_API_VERSION <= 17)
+        #undef S2N_AES_SHA1_COMPOSITE_AVAILABLE
+    #endif
+    /* Symbols for AES-SHA256-CBC composite ciphers were added in Openssl 1.0.2
  * See https://www.openssl.org/news/cl102.txt
  * These composite ciphers exhibit erratic behavior in LibreSSL releases.
  */
-#if S2N_OPENSSL_VERSION_AT_LEAST(1,0,2)
-#define S2N_AES_SHA256_COMPOSITE_AVAILABLE
-#endif
-#if defined(AWSLC_API_VERSION) && (AWSLC_API_VERSION <= 17)
-#undef S2N_AES_SHA256_COMPOSITE_AVAILABLE
-#endif
+    #if S2N_OPENSSL_VERSION_AT_LEAST(1, 0, 2)
+        #define S2N_AES_SHA256_COMPOSITE_AVAILABLE
+    #endif
+    #if defined(AWSLC_API_VERSION) && (AWSLC_API_VERSION <= 17)
+        #undef S2N_AES_SHA256_COMPOSITE_AVAILABLE
+    #endif
 #endif
 
 /* Silly accessors, but we avoid using version macro guards in multiple places */
 static const EVP_CIPHER *s2n_evp_aes_128_cbc_hmac_sha1(void)
 {
-    #if defined(S2N_AES_SHA1_COMPOSITE_AVAILABLE)
```

Comment:
Getting rid of the indent on `#if` could be controversial, although I'd think there is a way to ensure this in clang-format.